<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnvelopeLibraryWindow</class>
 <widget class="QMainWindow" name="EnvelopeLibraryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Envelope Library</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="leftWidget">
       <layout class="QVBoxLayout" name="leftLayout">
        <item>
         <widget class="QLineEdit" name="searchEdit">
          <property name="placeholderText">
           <string>Search envelopes...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="envelopeTreeView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="buttonLayout">
          <item>
           <widget class="QPushButton" name="newButton">
            <property name="text">
             <string>New</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="duplicateButton">
            <property name="text">
             <string>Duplicate</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rightWidget">
       <layout class="QVBoxLayout" name="rightLayout">
        <item>
         <widget class="QLabel" name="itemLabel">
          <property name="text">
           <string>Item View</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="itemViewWidget"/>
        </item>
        <item>
         <widget class="QWidget" name="drawingAreaWidget"/>
        </item>
        <item>
         <layout class="QGridLayout" name="valueLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="xLabel">
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="xValueEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="yLabel">
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="yValueEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEnvelope">
    <property name="title">
     <string>Envelope</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionDuplicate"/>
    <addaction name="actionDelete"/>
   </widget>
   <addaction name="menuEnvelope"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionDuplicate"/>
   <addaction name="actionDelete"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDuplicate">
   <property name="text">
    <string>Duplicate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EnvLibDrawingArea</class>
   <extends>QWidget</extends>
   <header>EnvLibDrawingArea.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui> 